src/app/_services/api/accounts/reset-password.service.ts

```diff
@@ -14,7 +14,8 @@ export class ResetPasswordService {
     }
 
     putPasswordReset(input: {
```

Comment:
Ok yes. This is the change password that we already have and we want to keep this. This is used when the user knows their password and can login and want to change their password.
Your feature should be a seperate one.